actionview/lib/action_view/template.rb

```diff
@@ -261,11 +266,11 @@ def encode!
     # to ensure that references to the template object can be marshalled as well. This means forgoing
     # the marshalling of the compiler mutex and instantiating that again on unmarshalling.
     def marshal_dump # :nodoc:
-      [ @source, @identifier, @handler, @compiled, @original_encoding, @locals, @virtual_path, @updated_at, @format, @variant ]
+      [ @source, @identifier, @handler, @compiled, @locals, @virtual_path, @updated_at, @format, @variant ]
     end
 
     def marshal_load(array) # :nodoc:
-      @source, @identifier, @handler, @compiled, @original_encoding, @locals, @virtual_path, @updated_at, @format, @variant = *array
+      @source, @identifier, @handler, @compiled, @locals, @virtual_path, @updated_at, @format, @variant = *array
```

Comment:
Should we handle the case were people to marshal load an object created before this change?

Reply:
I don't think so.  I think the reason we introduced the marshal stuff is for parallel testing ([according to this comment](https://github.com/rails/rails/blob/082130d1d43620ca29c2f2f169a5059fa0065a8c/actionview/lib/action_view/template.rb#L265-L267).  Those shouldn't stick around.

Reply:
👍